docs/pages/modules/get-started.mdx

```diff
@@ -66,32 +75,29 @@ To run your app locally, run the `prebuild` command and then compile the app:
 
 Congratulations! You have created a local Expo module. You can now start working on it.
 
-> **Note:**
-> You can also use absolute import paths [with some configuration changes](https://expo.fyi/absolute-path-expo-modules.md).
+> **Note**: You can also use absolute import paths [by applying these configuration changes](https://expo.fyi/absolute-path-expo-modules.md).
 
 </Step>
 
 <Step label="3">
 
 ### Editing the module
 
-#### Android
-
-If you have an **android** directory in your project that you created using `npx expo prebuild`, you can open the directory in Android Studio.
+To test and develop your module locally, use `npx expo prebuild` to generate the native project directories and use `npx expo run:android` or `npx expo run:ios` to compile and run the example app instead of using EAS Build.
 
-You can always just edit the files in the `modules/my-module/android/src/main/java/expo/modules/mymodule/` directory directly in your favorite text editor.
+#### Android
 
-Change the `hello` method to return a different string. For example, you can change it to return "Hello world! 🌎🍎".
+If you have an **android** directory in your project that is generated by `npx expo prebuild`, you can open the project directory in Android Studio. Then, you will be able to edit the native code in the **modules/my-module/android/src/main/java/expo/modules/mymodule/** directory.
 
-Rebuild the app or build a new development client and you should see your change.
+For example, change the `hello` method to return a different string such as "Hello world! 🌎🤖". Then, run `npx expo prebuild` to generate the native directory and use `npx expo:android` to run the example app.
 
 #### iOS
 
-Open the files in **modules/my-module/ios/** directory in your favorite code editor to edit them. Alternatively, if you have an **ios** directory in your project that was created using `npx expo prebuild`, you can use Xcode to edit them.
+If you have an **ios** directory in your project that is generated by `npx expo prebuild`, open the project's workspace (**ios/your-app.workspace**) in Xcode. Then, you will be able to edit the native code in **Pods** > **Development Pods** > **MyModule**.
 
-Now, change the `hello` method to return a different string. For example, you can change it to return "Hello world! 🌎🍎".
+For example, change the `hello` method to return a different string such as "Hello world! 🌎🍎". Then, run `npx expo prebuild` to generate the native directory and use `npx expo:ios` to run the example app.
 
-Rebuild the app or build a new development client and you should see your change. Remember you need to either run `npx expo prebuild` each time you make a native change or reinstall the pods using `pod install --project-directory="example/ios"` (which should be way faster).
+Remember you need to either run `npx expo prebuild` each time you make a native change or reinstall the pods using `pod install --project-directory="ios"` (which will be faster).
```

Comment:
It's not necessary to run `prebuild` after each native change. You only need to `run` it again.

`pod install` might be necessary when you modify `expo-module.config.json` (e.g. add another module class) and when you add more files to the module (they won't appear in Xcode if you add them using something else than Xcode).